pyaedt/edb_grpc/core/edb_data/hfss_simulation_setup_data.py

```diff
@@ -0,0 +1,2146 @@
+
+    @defeature_abs_length.setter
+    def defeature_abs_length(self, value):
+        self._defeature_settings.DefeatureAbsLength = value
+        self._parent._update_setup()
+
+    @property
+    def defeature_ratio(self):
+        """Defeature ratio.
+
+        Returns
+        -------
+        float
+        """
+        return self._defeature_settings.DefeatureRatio
+
+    @defeature_ratio.setter
+    def defeature_ratio(self, value):
+        self._defeature_settings.DefeatureRatio = value
+        self._parent._update_setup()
+
+    @property
+    def healing_option(self):
+        """Whether to turn on healing of mis-aligned points and edges.
+        Options are:
+        0- Turn off.
+        1- Turn on.
+
+        Returns
+        -------
+        int
+        """
+        return self._defeature_settings.HealingOption
+
+    @healing_option.setter
+    def healing_option(self, value):
+        self._defeature_settings.HealingOption = value
+        self._parent._update_setup()
+
+    @property
+    def model_type(self):
+        """Model type.
+        Options:
+        0- General.
+        1- IC.
+
+        Returns
+        -------
+        int
+        """
+        return self._defeature_settings.ModelType
+
+    @model_type.setter
+    def model_type(self, value):
+        """Model type (General 0 or IC 1)."""
+        self._defeature_settings.ModelType = value
+        self._parent._update_setup()
+
+    @property
+    def remove_floating_geometry(self):
+        """Whether to remove floating geometries.
+
+        Returns
+        -------
+        bool
+            ``True`` if floating geometry removal is used, ``False`` otherwise.
+        """
+        return self._defeature_settings.RemoveFloatingGeometry
+
+    @remove_floating_geometry.setter
+    def remove_floating_geometry(self, value):
+        self._defeature_settings.RemoveFloatingGeometry = value
+        self._parent._update_setup()
+
+    @property
+    def small_void_area(self):
+        """Small voids to remove area.
+
+        Returns
+        -------
+        float
+        """
+        return self._defeature_settings.SmallVoidArea
+
+    @small_void_area.setter
+    def small_void_area(self, value):
+        self._defeature_settings.SmallVoidArea = value
+        self._parent._update_setup()
+
+    @property
+    def union_polygons(self):
+        """Whether to turn on the union of polygons before meshing.
+
+        Returns
+        -------
+        bool
+            ``True`` if union polygons is used, ``False`` otherwise.
+        """
+        return self._defeature_settings.UnionPolygons
+
+    @union_polygons.setter
+    def union_polygons(self, value):
+        self._defeature_settings.UnionPolygons = value
+        self._parent._update_setup()
+
+    @property
+    def use_defeature(self):
+        """Whether to turn on the defeature.
+
+        Returns
+        -------
+        bool
+            ``True`` if defeature is used, ``False`` otherwise.
```

Comment:
```suggestion
        """Whether to turn on the defeature.

        Returns
        -------
        bool
            ``True`` if defeature is used, ``False`` otherwise.
```
```suggestion
        """Whether defeature is used.

        Returns
        -------
        bool
            ``True`` if defeature is used, ``False`` otherwise.
```